R/McmcOptions-class.R

```diff
@@ -44,20 +57,28 @@ NULL
 #' @param step (`count`)\cr only every step-th iteration is saved after
 #'   the burn-in.
 #' @param samples (`count`)\cr number of resulting samples.
+#' @param RNG_kind (`string`)\cr the name of the RNG type. Possible types are:
+#'   `base::Wichmann-Hill`, `base::Marsaglia-Multicarry`, `base::Super-Duper`,
+#'   `base::Mersenne-Twister`.
+#' @param RNG_seed (`number`)\cr RNG seed as integer value.
 #'
 #' @export
 #' @example examples/McmcOptions-class-McmcOptions.R
 #'
 McmcOptions <- function(burnin = 1e4L,
                         step = 2L,
-                        samples = 1e4L) {
+                        samples = 1e4L,
+                        RNG_kind = "base::Mersenne-Twister",
+                        RNG_seed = 1L) {
```

Comment:
generally here please be careful not to use uppercase in the variable/slot names. so `rng_kind` and `rng_seed` .